Error compiling source on MacOS Catalina

Hi, cannot build from source on macos catalina:

	gcc -DHAVE_CONFIG_H -I. -I..  -D_USE_AUTOCONF -D_GNU_SOURCE -I../include -I.  -I/usr/local/include/SDL -D_GNU_SOURCE=1 -D_THREAD_SAFE   -DUSE_NETWORK  -DHAS_YM3812=1 -DHAS_Y8950=0 -DHAS_YM3526=0    -g -O2 -MT fmt/mod.o -MD -MP -MF $depbase.Tpo -c -o fmt/mod.o ../fmt/mod.c &&\
	mv -f $depbase.Tpo $depbase.Po
../fmt/mod.c:112:9: error: implicit declaration of function 'fmt_mod_load_song' is invalid in C99
      [-Werror,-Wimplicit-function-declaration]
        return fmt_mod_load_song(song, fp, lflags, 0);
               ^
../fmt/mod.c:112:9: note: did you mean 'fmt_mid_load_song'?
../include/fmt-types.h:100:16: note: 'fmt_mid_load_song' declared here
READ_INFO(mid) LOAD_SONG(mid)
               ^
../include/fmt.h:84:37: note: expanded from macro 'LOAD_SONG'
#define LOAD_SONG(t)            int fmt_##t##_load_song         PROTO_LOAD_SONG;
                                    ^
<scratch space>:132:1: note: expanded from here
fmt_mid_load_song
^
../fmt/mod.c:119:9: error: implicit declaration of function 'fmt_mod_load_song' is invalid in C99
      [-Werror,-Wimplicit-function-declaration]
        return fmt_mod_load_song(song, fp, lflags, 1);
               ^
2 errors generated.
make[1]: *** [fmt/mod.o] Error 1
make: *** [all] Error 2

--- a/fmt/mod.c
+++ b/fmt/mod.c
@@ -105,23 +105,9 @@ int fmt_mod_read_info(dmoz_file_t *file, const uint8_t *data, size_t length)
 
 /* --------------------------------------------------------------------------------------------------------- */
 
-/* loads everything but old 15-instrument mods... yes, even FLT8 and WOW files
-   (and the definition of "everything" is always changing) */
-int fmt_mod31_load_song(song_t *song, slurp_t *fp, unsigned int lflags)
-{
-	return fmt_mod_load_song(song, fp, lflags, 0);
-}
-
-/* loads everything including old 15-instrument mods. this is a separate
-   function so that it can be called later in the format-checking sequence. */
-int fmt_mod15_load_song(song_t *song, slurp_t *fp, unsigned int lflags)
-{
-	return fmt_mod_load_song(song, fp, lflags, 1);
-}
-
 /* force determines whether the loader will force-read untagged files as
    15-sample mods */
-int fmt_mod_load_song(song_t *song, slurp_t *fp, unsigned int lflags, int force)
+static int fmt_mod_load_song(song_t *song, slurp_t *fp, unsigned int lflags, int force)
 {
 	uint8_t tag[4];
 	int n, npat, pat, chan, nchan, nord;
@@ -397,3 +383,16 @@ int fmt_mod_load_song(song_t *song, slurp_t *fp, unsigned int lflags, int force)
 	return LOAD_SUCCESS;
 }
 
+/* loads everything but old 15-instrument mods... yes, even FLT8 and WOW files
+   (and the definition of "everything" is always changing) */
+int fmt_mod31_load_song(song_t *song, slurp_t *fp, unsigned int lflags)
+{
+	return fmt_mod_load_song(song, fp, lflags, 0);
+}
+
+/* loads everything including old 15-instrument mods. this is a separate
+   function so that it can be called later in the format-checking sequence. */
+int fmt_mod15_load_song(song_t *song, slurp_t *fp, unsigned int lflags)
+{
+	return fmt_mod_load_song(song, fp, lflags, 1);
+}

